packages/mui-base/src/useInput/useInput.types.ts

```diff
@@ -16,7 +16,7 @@ export interface UseInputParameters {
    * The prop defaults to the value (`false`) inherited from the parent FormControl component.
    */
   error?: boolean;
-  onBlur?: React.FocusEventHandler;
+  onBlur?: React.FocusEventHandler<HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
Should other types in this file that use `HTMLInputElement`, for example `onChange` and `inputRef` a couple lines below, also have `HTMLInputElement | HTMLTextAreaElement` type instead?

Reply:
Makes sense ~ I updated it for the relevant handlers and ref